CHANGES.md

```diff
@@ -1,3 +1,9 @@
+0.7.2.0
```

Comment:
Could you please remove `.0` from version here and from `.cabal` file? Workflow to our project suggests to use only 3 numbers in version. But sometimes I pretend that I'm not aware of this rule and in case of documentation changes it's ok to add 4th number. But in general we should follow our style-guide...

Reply:
Ok, sure!